tests/components/axis/conftest.py

```diff
@@ -42,7 +46,6 @@
 )
 
 from tests.common import MockConfigEntry
-from tests.components.light.conftest import mock_light_profiles  # noqa: F401
```

Comment:
cruft?

Reply:
I didn't remember why I needed it and I've seen other things no longer relevant so I just tried to remove it and nothing broke :)